content/en/docs/reference/setup-tools/kubeadm/kubeadm-join.md

```diff
@@ -12,7 +12,7 @@ This command initializes a Kubernetes worker node and joins it to the cluster.
 <!-- body -->
 {{< include "generated/kubeadm_join.md" >}}
 
-### The join workflow {#join-workflow}
+### The join workflow
```

Comment:
check the 'kubeadm init' page for anchor formatting.
i don't think we should be removing these because they might be linked from other k8s.io pages or externally.

we should have some anchors.

Reply:
Fix it,Thanks for your advice.